### DISCUSSION TITLE
Bug: Dark mode not working properly when creating an organization

### DISCUSSION THREAD

**Discussion by 0xdsm** [community/community#19936] [Projects and Issues]
Hello, Github Community! Hope everyone is doing well
I want to report a bug on GitHub dark mode that happens when I access the creating organization function.

If we click on our profile picture on the top bar and click in the button "Your organization" everything will loads correctly and a bunch of options appear to be used, ok everything is fine.

![image](https://user-images.githubusercontent.com/50470310/177043266-256ab04d-819a-4336-a4d2-6d0c9148fae6.png)

But when I click in "New Organization" button the dark mode disappear and all the page becomes white

![image](https://user-images.githubusercontent.com/50470310/177043336-8da3dfc6-3ad5-456e-ae4a-900d2a0765ba.png)
![image](https://user-images.githubusercontent.com/50470310/177043345-6315a22c-278d-407e-b3ed-0316ef302d0f.png)

Literally a flashlight! But ok, I give up of the idea of creating an organization and clicked in the back to the last page icon on my browser, when I go back the Your Organizations page is on the white mode too.

![image](https://user-images.githubusercontent.com/50470310/177043413-27805f27-7e88-494d-99e1-a3d7b3750074.png)

I need to refresh the page or access other functions to GitHub backs to the Dark Mode. That's probably a CSS or JS loading error and I think it's extremely easy to fix. The same things happens on other browsers, I've tested on my Chrome.

*And sorry for some english typo, I'm not english native ;)*

Best regards,
oppsec.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
